geode-core/src/main/java/org/apache/geode/cache/query/internal/ExecutionContext.java

```diff
@@ -603,10 +621,6 @@ int nextFieldNum() {
     throw new UnsupportedOperationException("Method should not have been called");
   }
 
-  public void setCqQueryContext(boolean cqQuery) {
-    throw new UnsupportedOperationException("Method should not have been called");
-  }
-
```

Comment:
why is it OK to remove this method?

Reply:
Because we set it in the constructor now